### ISSUE TITLE
Trivial typo in universe/project/self-attentive-parser

### ISSUE THREAD

**Issue by forest1988** [explosion/spaCy#6424]
## How to reproduce the behaviour

I tried to use `self-attentive-parser` in The spaCy Universe.
However, I got a SyntaxError when I run the example code.

The error occurs In line 6, 

```
    doc = nlp('The time for action is now. It's never too late to do something.')
                                              ^
SyntaxError: invalid syntax
```

I think the problem is that the sentence enclosed in `' '` has `It's` and contains `'`.
I've checked the JSON file that contains the contents of Universe, and found that the line contains the sentence is surrounded further out with `" "`.

https://github.com/explosion/spaCy/blob/51232ffb9e20bec522a12a3da720efee4097cd08/website/meta/universe.json#L1686

By the way, is here the correct place to report any bugs I find in the spaCy Universe and its documentation? 
Or should I contact the author directly by referring to the author info in the Universe's documentation?
If here is not the right place, I apologize for the misunderstanding.

Thank you in advance.

## Your Environment

* **spaCy version:** 2.3.2
* **Platform:** Linux-4.15.0-123-generic-x86_64-with-glibc2.10
* **Python version:** 3.8.3

**Comment by svlandeg** [comment]
Hi! Thanks for the report - we should indeed fix this. If you find general bugs in a package in the spaCy universe, you can contact the authors directly on their github tracker. But in this case, the issue is with the documentation page of spaCy, so it's fine to report here. I think we can just avoid the issue of quotes in quotes in quotes, and write "It is" instead ;-)

Do you want to make the PR?

**Comment by forest1988** [comment]
Thank you for your kind and detailed response!
I now understood that if we find general bugs in the spaCy universe, I can contact the authors directly. I also understood that when it is about the problems with the documentation page of spaCy, it's fine to report here.

Thank you for telling me how to avoid the issue! I will try it soon, and if you don't mind, I'd love to make the PR.

**Comment by svlandeg** [comment]
Great - feel free to go ahead!

**Comment by svlandeg** [comment]
Fixed in PR #6428 - thanks for the report and PR!

**Comment by forest1988** [comment]
Thank you again for your kind response!
